### PR TITLE
added expandable search-box for search on contents

### PR DESCRIPTION
**Pull request by architkshk**
#205 https://github.com/coding-blocks/codingblocks.online.projectx/issues/131
@vdvibhu20 @witty123 
screenshot from 2018-07-12 18-19-41
screenshot from 2018-07-12 18-22-39
screenshot from 2018-07-12 18-22-47

### CODE REVIEW

**Comment by vdvibhu20** [comment]
There is no need to create a new object `o-expand-search` use `input-search` and `search-box` to create search input and search result respectively. 
To hide search result use `display-none` class on search-box.
@architkshk you can refer to nav-bar component to see input-search and result markup

**Comment by architkshk** [comment]
@vdvibhu20 what do I do to add the transition and changing Color of the search box?

**Comment by vdvibhu20** [comment]
you can see the list of background colours in elements/elements.bg-color.scss and select one from there or add one with an appropriate name.
For transition you can add a class and overwrite it in object layer.

**Comment by architkshk** [comment]
@vdvibhu20 expand-search class adds the properties required for the animation and the colours are in settings.color.scss. Please elaborate a little more what changes are extra or what should i change.


peek 2018-07-19 23-56

**Comment by vdvibhu20** [comment]
@architkshk as of now there is no need to add a transition to the search box as you can see on the home page. This will ensure uniformity in overall design. You can get background colours from class name [`bg-colorname`](https://github.com/coding-blocks/motley/blob/master/sass/styles/elements/elements.bg-color.scss).

**Comment by architkshk** [comment]
@vdvibhu20 Done

**Comment by architkshk** [comment]
@vdvibhu20 updates?

File: sass/styles/elements/elements.bg-color.scss [hunk 1]
```diff
@@ -2,7 +2,9 @@ $list:  (red, $fail-red)
         (black, $global-font-text-color)
         (white, $white )
         (grey, $nav-border)
-        (green, $font-complete);
+        (green, $font-complete)
+        (grey2, $expand-search-bg)
```
Comment (witty123) [inline sass/styles/elements/elements.bg-color.scss]: please use proper names for variables

Comment (architkshk) [reply sass/styles/elements/elements.bg-color.scss]: Please suggest how should i name variants of a color (here grey).

**Comment by architkshk** [comment]
@witty123 is this fine now?

**Pull request closed without merging**